### DISCUSSION TITLE
User table column not updating

### DISCUSSION THREAD

**Discussion by m7kael** [Budibase/budibase#7662] [Help] [answered]
Hi, 
I am working on a customer portal and I have a column in the User table with a customer ID, the value in that column is used when quering data with rest API from a external data source. I add users with the budibase api and then update the row in the users table with customer ID to the external datasource. My issue is that in order to get the {{ Current User.CustomerId }} binding to work I have to publish the app manually after everytime I add a new user. Else, ther binding {{ Current User.CustomerId }} returns empty even if the column in the user table is populated with correct value. Tried researching the forum but could not find any other discussions regarding this. Any advice?

**Comment by andz-bb** [comment, marked as answer]
Hey @m7kael, I think this one could be quite similar to #7649.

There are two DBs here, dev and production, and the Users table from dev only syncs over to production when you publish. Is it possible when you hit the Budibase API it's only changing the user data on the dev table? I think the best thing to check is the app ID on your requests, does the ID include `_dev`?

**Reply by m7kael** [reply]
Hi,
Yes, that was the problem, thank you for you help!
